### PR TITLE
Issue # 3, added person's biometric specific attributes.

### PR DESCRIPTION
**Pull request by mahammedtaheer**
Adds person's individual biometrics to different claim ids. 
Adds support for templates for all the modalities.

Resolves https://github.com/mosip/id-claim-169/issues/3

### CODE REVIEW

File: draft-sasikumar-id-claim-169.md [hunk 1]
```diff
@@ -70,57 +70,228 @@ Claim 169 represents a JSON Object that includes the below table as ID attribute
...
+| `1`       | JPEG        |
+| `2`       | JPEG2000    |
+| `3`       | AVIF        |
+| `4`       | WEBP        |
+| `5`       | TIFF        |
+| `6`       | WSQ         |
+
+###### Template
+
+| Subformat  | Description                      |
```
Comment (gsasikumar) [inline draft-sasikumar-id-claim-169.md:179]: I think there are more template standards in CBEF or ISO

File: draft-sasikumar-id-claim-169.md [hunk 2]
```diff
@@ -70,57 +70,228 @@ Claim 169 represents a JSON Object that includes the below table as ID attribute
...
+  10: New City, METRO LINE, PA # Address
+  11: peter@example.com # Email ID
+  12: "+1 234-567" # Phone number
+  13: US # Nationality
+  14: 2 # Marital status: Married
+  15: Jhon Honai # Guardian
+  16: 03CBABDF83D068ACB5DE65B3CDF25E0036F2C546CB90378C587A076E7A759DFD27CA7872B6CDFF339AEAACA61A6023FD1D305A9B4F33CAA248CEDE38B67D7C915C59A51BB4E77D10077A625258873183F82D65F4C482503A5A01F41DEE612C3542E5370987815E592B8EA2020FD3BDDC747897DB10237EAD179E55B441BC6D8BAD07CE535129CF8D559445CC3A29D746FBF1174DE2E7C0F3439BE7DBEA4520CF88825AAE6B1F291A746AB8177C65B2A459DD19BD32C0C3070004B85C1D63034707CC690AB0BA023350C8337FC6894061EB8A714A8F22FE2365E7A904C72DEC9746ABEA1A3296ECACD1A40450794EDCD2B34844E7C19EB7FB1A4AF3B05C3B374BD2941603F72D3F9A62EAB9A2FDAEEEEC8EE6E350F8A1863C0A0AB1B4058D154559A1CD5133EFCF682ABC339960819C9427889D60380B635A7D21D017974BBA57798490F668ADD86DA58125D9C4C1202CA1308F7734E43E8F77CEB0AF968A8F8B88849F9B98B26620399470ED057E7931DED82876DCA896A30D0031A8CBD7B9EDFDF16C15C6853F4F8D9EEC09317C84EDAE4B349FE54D23D8EC7DC9BB9F69FD7B7B23383B64F22E25F # Binary image
+  17: 2 # Binary image format: JPEG
+  18: [1, 2] # Best quality fingers
+  50: # Right Thumb Biometrics
```
Comment (sree96) [inline draft-sasikumar-id-claim-169.md:219]: This should be a [Biometrics] right. The example representations looks different. Can we change that?

**Comment by nb-tech5** [comment]
Apart from the examples that need to be updated, I'm ok with the changes proposed regarding the support for multi-modality and biometric templates.

File: draft-sasikumar-id-claim-169.md (hunk 1 above)
Comment (mahammedtaheer) [reply draft-sasikumar-id-claim-169.md:179]: Mentioned template standards are the most commonly used. We definitely have the option to extend this list in future if we want to add more template standards.

File: draft-sasikumar-id-claim-169.md (hunk 2 above)
Comment (mahammedtaheer) [reply draft-sasikumar-id-claim-169.md:219]: In yaml format, the array are allowed to represent using braces [] or with new line and tab separator. As we have more attributes in attribute so specified with new line.

**Review by sree96 — APPROVED** [review]

File: draft-sasikumar-id-claim-169.md [hunk 3]
```diff
@@ -70,57 +70,228 @@ Claim 169 represents a JSON Object that includes the below table as ID attribute
...
+| `16`      | `tstr`         | Binary Image         | Binary image of the person's photograph                                                                                                                                                                                                 |
+| `17`      | `int`          | Binary Image Format  | Binary image format. Can contain the following values `1` - JPEG, `2` - JPEG2, `3` - AVIF, `4` - WEBP                                                                                                                                   |
+| `18`      | `[int]`        | Best Quality Fingers | An unsigned 8-bit number encoding the hand position of the finger. It must be in the range 0-10, where 0 represents "Unknown", 1-5 represents right thumb to little finger, and 6-10 represents left thumb to little finger in sequence |
+| `19.. 49` |                | Reserved             | Reserved for future attributes                                                          |
+
```
Comment (nb-tech5) [inline draft-sasikumar-id-claim-169.md:94]:
```suggestion
```
Extra new line is breaking the table

File: draft-sasikumar-id-claim-169.md [hunk 4]
```diff
@@ -70,57 +70,228 @@ Claim 169 represents a JSON Object that includes the below table as ID attribute
...
+| Subformat | Description |
+|-----------|-------------|
+| `0`       | PNG         |
+| `1`       | JPEG        |
+| `2`       | JPEG2000    |
+| `3`       | AVIF        |
+| `4`       | WEBP        |
+| `5`       | TIFF        |
+| `6`       | WSQ         |
```
Comment (nb-tech5) [inline draft-sasikumar-id-claim-169.md:175]:
```suggestion
| `6`       | WSQ         |
| `100..200` | Vendor specific                  |
```
Here will be useful to also have a vendor specific region, we already have that use case.

File: draft-sasikumar-id-claim-169.md [hunk 5]
```diff
@@ -70,57 +70,228 @@ Claim 169 represents a JSON Object that includes the below table as ID attribute
...
+
+    # Voice sound
+    - 0: 03CBA(...)0378C58 # Data
+      2: 1 # MP3
+  
+    # Voice template
+    - 0: 03CBA(...)0378C58 # Data
+      1: 1 # Template
+      2: 100 # Vendor specific
+      3: VendorZ # Biometric data issuer
```
Comment (nb-tech5) [inline draft-sasikumar-id-claim-169.md:294]: There're some whitespace mis-alignments (YAML specific format requires whitespace alignment between siblings), repeated iris field number and add some comments to keep them normalized. 

```suggestion
  50: # Right Thumb Biometrics

    # Right Thumb image
    - 0: 03CBA(...)0378C58 # Data
      1: 0 # Image
      2: 1 # JPEG

    # Right Thumb template
    - 0: 03CBA(...)0378C58 # Data
      1: 1 # Template
      2: 100 # Vendor specific
      3: VendorA # Biometric data issuer

  51: # Right Pointer Finger Biometrics

    # Right Pointer Finger image
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Image
      2: 6 # WSQ
      3: VendorA # Biometric data issuer

    # Right Pointer Finger template
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Template
      2: 1 # Fingerprint Template ISO 19794-2
      3: VendorA # Biometric data issuer

  58: # Left Ring Finger Biometrics

    # Left Ring Finger image
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Image
      2: 6 # WSQ
      3: VendorA # Biometric data issuer

    # Left Ring Finger template
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Template
      2: 1 # Fingerprint Template ISO 19794-2
      3: VendorB # Biometric data issuer

  60: # Right Iris Biometrics

      # Right Iris image
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Image
      2: 6 # WSQ
      3: VendorX # Biometric data issuer

    # Right Iris image
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Image
      2: 6 # WSQ
      3: VendorY # Biometric data issuer

  61: # Left Iris Biometrics

    # Left Iris templates
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Template
      2: 100 # Vendor specific
      3: VendorX # Biometric data issuer
   
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Template
      2: 100 # Vendor specific
      3: VendorY # Biometric data issuer

  65: # Voice Biometrics

    # Voice sound
    - 0: 03CBA(...)0378C58 # Data
      2: 1 # MP3
  
    # Voice template
    - 0: 03CBA(...)0378C58 # Data
      1: 1 # Template
      2: 100 # Vendor specific
      3: VendorZ # Biometric data issuer
```

**Review by nb-tech5 — CHANGES_REQUESTED** [review]

**Comment by nb-tech5** [comment]
Hi, @mahammedtaheer, let me know if I can help in any way to close the PR.

File: draft-sasikumar-id-claim-169.md (hunk 3 above)
Comment (mahammedtaheer) [reply draft-sasikumar-id-claim-169.md:94]: removed the extra new line.

File: draft-sasikumar-id-claim-169.md (hunk 5 above)
Comment (mahammedtaheer) [reply draft-sasikumar-id-claim-169.md]: removed and formatted correctly.

**Pull request merged**